pkg/apis/networking/v1beta1/conversion.go

```diff
@@ -55,3 +55,8 @@ func Convert_networking_IngressSpec_To_v1beta1_IngressSpec(in *networking.Ingres
 	}
 	return nil
 }
+
+func Convert_networking_HTTPIngressPath_To_v1beta1_HTTPIngressPath(in *networking.HTTPIngressPath, out *v1beta1.HTTPIngressPath, s conversion.Scope) error {
+	// this drops the new PathType field as there is no equivalent in v1beta1.
+	return autoConvert_networking_HTTPIngressPath_To_v1beta1_HTTPIngressPath(in, out, s)
```

Comment:
This is not OK - you need to retain this information for round-trip.

Reply:
@robscott I'm pretty sure you can replicate these errors with the following

```
~/go/src/k8s.io/kubernetes$ go test ./pkg/api/testing/...
```

Reply:
Not sure how I missed/broke all the round tripping here, should all be fixed now. Thanks for the helpful test example @cmluciano!